### ISSUE TITLE
Should we also add another section to indicate beginner, intermediate or expert?

### ISSUE THREAD

**Issue by anbuselvan** [lpatmo/cb-links#13]
It would be nice to select the level as well along with free or paid when posting links.

**Comment by colbycheeze** [comment]
Free or paid is a great tag to have. I never know what "beginner, intermediate, expert" ever means when reading levels on a tutorial. That is always so relative to opinion.

I don't know the solution to that, but I wonder if there is a better way to relate what level someone should be at before attempting a tutorial of sorts...

Maybe if we have some trails or checkpoint type of resources for various things...like say for example we created a cb-links resource that said "If you are learning HTML, here are your checkpoints..." and then we say beginner should be able to answer  1-10, intermediate can identify and answer 1-50, and expert level would be beyond that?

Anyway, that's my disorganized thoughts on the subject. What do you think?

**Comment by lpatmo** [comment]
Ohh, like a set of questions that we'd offer as a guide for tagging resources?

**Comment by anbuselvan** [comment]
@colbycheeze yes, I totally agree with you, and I'm still trying to understand your recommendations on checkpoints. @lpatmo yes, more tagging would be nice to organize and group things get the relevant resources quickly, just like labels from github.
